src/app/api.ts

```diff
@@ -1944,6 +1963,27 @@ export class RunTaskResponse {
   endTime?: Date;
 }
 
+@jsonObject
+export class UserOrgResponse {
```

Comment:
There's already OrgResponse that it's the same value used also by the backend. So this type isn't needed.

Looks like to current OrgResponse has a wrong field called `username` instead of `name` (that should be fixed, and currently it isn't used).

Reply:
> There's already OrgResponse that it's the same value used also by the backend. So this type isn't needed.
> 
> Looks like to current OrgResponse has a wrong field called `username` instead of `name` (that should be fixed, and currently it isn't used).

Ok I'll fix it.